OpenEdXMobile/res/values/strings.xml

```diff
@@ -195,6 +195,10 @@
     <string name="label_ended">Ended - {date}</string>
     <!-- Label for date when course is ending -->
     <string name="label_ending">Ending - {date}</string>
+    <!--Label for course that has not downloaded yet!-->
+    <string name="label_download_media">Save Course Media for Offline View</string>
```

Comment:
Add tools:ignore="MissingTranslation" tag in it to mention you have not translated it for other languages.